### ISSUE TITLE
Error compiling version 1.1.4 using Cocoapods

### ISSUE THREAD

**Issue by andreaslindahl** [syedhali/EZAudio#267]
I updated to version 1.1.4 (from 1.1.2) using Cocoapods and am now getting an error when building:

"'EZAudioiOS/EZAudio.h' file not found" in EZAudioiOS.h

**Comment by syedhali** [comment]
What version of Xcode and cocoapods are you running?

**Comment by andreaslindahl** [comment]
Cocoapods 0.39.0 and XCode 7.2

**Comment by syedhali** [comment]
Can you just include it like:
#import <EZAudio/EZAudio.h>

**Comment by syedhali** [comment]
```
#import <EZAudio/EZAudio.h>
```

**Comment by andreaslindahl** [comment]
The error doesn't appear in any of my files, but in the EZAudioiOS.h file in the Cocoapods project.

**Comment by andreaslindahl** [comment]
EZAudioiOS.h isn't included in the Pod if I use EZAudio version 1.1.2

**Comment by andreaslindahl** [comment]
Seems like this file was changed for 1.1.3: https://github.com/syedhali/EZAudio/commit/470d3cdd8b84db9b25b7a8a3e87fd804034a8fbe.

But 1.1.3 doesn't seem to be available via Cocoapods, meaning I have used 1.1.2 up until now, since this is the first time I'm seeing this bug.

**Comment by fcjxxl** [comment]
I met the same problem, and I change EZAudioiOS.h and EZAudioOSX.h to #import "EZAudio.h", then it build succeeded

**Comment by andreaslindahl** [comment]
I changed it to make it compile, as well. But I would rather not edit the files in my Cocoapods project...

**Comment by cmmartin** [comment]
Same issue here. I just reverted to `1.1.2` for now...

`pod 'EZAudio', '1.1.2'`

**Comment by syedhali** [comment]
Hey everyone, please try again with 1.1.5

`pod 'EZAudio', '1.1.2'`

**Comment by fcjxxl** [comment]
still
/Pods/EZAudio/EZAudio/EZAudioiOS.h:26:9: 'EZAudioiOS/EZAudio.h' file not found

**Comment by andreaslindahl** [comment]
Nope, still doesn't work. It looks like you're trying to remove these files in the podspec file, but that doesn't seem to work...

**Comment by lluisgerard** [comment]
Same problem here with `1.1.5`, had to manually comment `EZAudioOSX.h` and `EZAudioiOS.h` imports to compile.

**Comment by fform** [comment]
Can confirm this with `1.1.5` and `1.1.2` using XCode `7.2`. Changing the path fixes it.

FWIW, I was using `1.1.2` before and it was working with XCode 7.2 but I had to clean out my Pod folder on a branch change. When I reinstalled `1.1.2` fresh, it no longer would compile.

**Comment by don-inkscreen** [comment]
Had to use lluisgerard's solution for 1.1.5. Cocoapods refused to install 1.1.2 for me.

**Comment by vinbhai4u** [comment]
I got it fixed by changing 

_`#import <EZAudioiOS/EZAudio.h>`_

to

**#import "EZAudio.h"**

in 

EZAudioiOS.h and EZAudioOSX.h

**Comment by brnunes** [comment]
Same problem on `1.1.4`, had to change `#import <EZAudioiOS/EZAudio.h>` to `#import "EZAudio.h"` in `EZAudioiOS.h` and `EZAudioOSX.h`

**Comment by codezero-jp** [comment]
I use [pod/diffs](https://github.com/jpsim/pod-diffs) to change 
`#import <EZAudioiOS/EZAudio.h> `
to 
`#import` `"EZAudio.h"`
 in EZAudioiOS.h and EZAudioOSX.h

**Comment by davidchappelle** [comment]
Seems like a super trivial thing to patch? can't we just get a working version that has the path fix that everyone is using above?

**Comment by abdultcb** [comment]
I got it fixed by changing

#import <EZAudioiOS/EZAudio.h>

to

#import "EZAudio.h"

in

EZAudioiOS.h and EZAudioOSX.h
even after updating to 1.1.5

**Comment by bideal** [comment]
@syedhali , thanks for your EZAudio project which saves me a lot of work, but please help solve this problem asap.

The error is obvious. the two files "**Pods/EZAudio/EZAudioiOS.h**" & "**Pods/EZAudio/EZAudioOSX.h**" that Cocoapods tries to build into the "**Products/EZAudio.framework**" import header files in this framework per se. 

Solution: please either remove these two header files or change the header reference in "*.h" format.

Thanks.

**Comment by liadgabriel** [comment]
Hi @syedhali, any news about is issue?
